### PR TITLE
Refactor of update_db arguments

### PR DESCRIPTION
**Pull request by kalisp**
## Changelog Description
Additional formatting refactor.

### CODE REVIEW

File: client/ayon_sitesync/addon.py [hunk 1]
```diff
@@ -492,19 +492,20 @@ def validate_project(self, project_name, site_name, reset_missing=False):
 
         sites_added = 0
         sites_reset = 0
-        for repre_entity in repre_entities:
+        repre_ids = [repre["id"] for repre in repre_entities]
+        repre_states = self.get_representations_sync_state(
+            project_name, repre_ids, site_name, site_name)
+
+        repres = get_representations(
```
Comment (iLLiCiTiT) [inline client/ayon_sitesync/addon.py:499]: Why you're requering them?

File: client/ayon_sitesync/addon.py [hunk 2]
```diff
@@ -492,19 +492,20 @@ def validate_project(self, project_name, site_name, reset_missing=False):
...
+            project_name, representation_ids=repre_ids
+        )
+        for repre_entity in repres:
```
Comment (iLLiCiTiT) [inline client/ayon_sitesync/addon.py:502]:
```suggestion
        for repre_entity in repre_entities:
```

File: client/ayon_sitesync/addon.py (hunk 1 above)
Comment (BigRoy) [reply client/ayon_sitesync/addon.py:499]: Haha, I was just about to ask! :)

File: client/ayon_sitesync/providers/abstract_provider.py [hunk 3]
```diff
@@ -76,7 +92,7 @@ def download_file(self, source_path, local_path,
             addon (SiteSyncAddon): addon instance to call update_db on
             project_name (str):
             file (dict): info about uploaded file (matches structure from db)
-            representation (dict): complete repre containing 'file'
+            repre_status (dict): complete repre containing 'file'
```
Comment (BigRoy) [inline client/ayon_sitesync/providers/abstract_provider.py:95]: Either the new argument name is odd or the description odd.

Also, I'm confused as to why the arguments in this docstring are not sorted in the order of the actual argument. It's to explain that the bottom ones are for saving progress - but do IDEs not get confused?

File: client/ayon_sitesync/addon.py (hunk 1 above)
Comment (kalisp) [reply client/ayon_sitesync/addon.py]: Removed

File: client/ayon_sitesync/providers/abstract_provider.py (hunk 3 above)
Comment (kalisp) [reply client/ayon_sitesync/providers/abstract_provider.py]: Updated.

Comment (iLLiCiTiT) [reply client/ayon_sitesync/providers/abstract_provider.py]: BTW the same text at multiple places.

Comment (kalisp) [reply client/ayon_sitesync/providers/abstract_provider.py]: Updated missed.

**Pull request merged**